concent_api/core/views.py

```diff
@@ -446,6 +487,18 @@ def handle_send_reject_report_computed_task(request, client_message):
     client_public_key = decode_client_public_key(request)
     validate_golem_message_reject(client_message.cannot_compute_task)
 
+    try:
+        subtask = Subtask.objects.get(
+            subtask_id                       = client_message.cannot_compute_task.task_to_compute.compute_task_def['subtask_id'],
+            report_computed_task__subtask_id = client_message.cannot_compute_task.task_to_compute.compute_task_def['subtask_id'],
+            state                            = Subtask.SubtaskState.FORCING_REPORT.name,  # pylint: disable=no-member
+            requestor__public_key            = request.META['HTTP_CONCENT_CLIENT_PUBLIC_KEY'],
+        )
+    except Subtask.DoesNotExist:
+        raise Http400("'ForceReportComputedTask' for this task and client has not been initiated yet. Can't accept your 'RejectReportComputedTask'.")
+
+    force_report_computed_task = deserialize_message(subtask.report_computed_task.data.tobytes())
+
     force_report_computed_task_from_database = StoredMessage.objects.filter(
         task_id                    = client_message.cannot_compute_task.task_to_compute.compute_task_def['task_id'],
         type                       = message.ForceReportComputedTask.TYPE,
```

Comment:
This is part of old logic and will be removed in in https://github.com/golemfactory/concent/issues/144. There was a reason for that in old logic, I can't remember exactly what was it, but you can ask @kbeker for details.

Reply:
@kbeker? At the very least the message was wrong.

Reply:
@cameel I have changed it to `<=2` when I was changing logic of creating new messages when client pinged receive end-point instead of giving it from database. If I remember corectly when I was changing it message `ForceReportComputedTaskResponse` didn't exist so I had to do it this way. My fault that I haven't changed message from assert. And excalty as @rwrzesien said this will be removed as a part of old logic.